spec/models/coach_spec.rb

```diff
@@ -1,5 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe Coach, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  def generate_code
+    return Array.new(6){rand 10}.join
```

Comment:
we should probably use ActiveRecords secure_token here: https://github.com/rails/rails/blob/master/activerecord/lib/active_record/secure_token.rb

Reply:
We may be thinking about two different attributes here. The code I'm thinking of is a six-digit code that a coach will give to a client to sign up. The code will appear on the coach's profile page and they will be responsible for giving that code to their client.

Reply:
I'll look into it